build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.evm-maven-publish.gradle.kts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import gradle.kotlin.dsl.accessors._c5ed198e191f30f39ee46d0abbbd888f.publishing
```

Comment:
This has to be removed. IntelliJ sometimes adds this bogus imports when it is not completely in sync with Gradle. Also (because of the import that was above the license header) Spotless added a second header that should be removed as well.